wrangler/deploy.mjs

```diff
@@ -11,7 +11,7 @@ import path from "path";
 import {
     Keypair
 } from "stellar-base";
-import * as TOML from "@iarna/toml";
```

Comment:
Is this an update? If so we should ensure we remove `toml` from the package.json. If this wasn't intentional let's revert.